xwiki-platform-core/xwiki-platform-oldcore/src/test/java/com/xpn/xwiki/pdf/impl/PdfExportImplTest.java

```diff
@@ -109,23 +113,24 @@ public void applyCSSWhenExistingStyleDefinedUsingShorthandNotation() throws Exce
 
         String expected = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
             + "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" "
-                + "\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">"
-                + "<html xmlns=\"http://www.w3.org/1999/xhtml\"><head><title>\n"
+            + "\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">"
+            + "<html xmlns=\"http://www.w3.org/1999/xhtml\"><head>\n"
+            + "<title>\n"
```

Comment:
indentation :)

Reply:
The previous indentation was arbitrary. Why indent `<html` more than its descendants? Makes no sense to me, and the original document had the same indentation there (so indentation is also broken on line 81 which I did not write).

Reply:
What I did was indent when it was supposed to be text on the same line. It wasn't arbitrary ;)

Reply:
> It wasn't arbitrary ;)

Well if the indentation criteria is based on element content whitespace (which is quite arbitrary in the original document) then it is. The DOCTYPE originally had a newline after it, which is lost in the process. Do I change that indent too?

Reply:
Don't worry I'm on it! Thx